src/components/utils/test/posix_thread_test.cc

```diff
@@ -32,6 +32,8 @@
 
 #include "gtest/gtest.h"
 #include "utils/lock.h"
+#define private public  // Need to be able to create an instance the Thread
```

Comment:
@LuxoftAKutsan @SNiukalov hot sure if it's a safe method to create the Thread instance, how do you think?

Reply:
@LitvinenkoIra @LuxoftAKutsan 
Fixed in:https://github.com/LuxoftSDL/sdl_core/pull/50/commits/ca62978ee665d27ecfceffac8caf96ce740ecb51